plugins/module_utils/scaleway.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, message):
 R_LINK_HEADER = r'''<[^>]+>;\srel="(first|previous|next|last)"
     (,<[^>]+>;\srel="(first|previous|next|last)")*'''
 # Specify a single relation, for iteration and string extraction purposes
-R_RELATION = r'<(?P<target_IRI>[^>]+)>; rel="(?P<relation>first|previous|next|last)"'
+R_RELATION = r'<\/(?P<target_IRI>[^>]+)>; rel="(?P<relation>first|previous|next|last)"'
```

Comment:
```suggestion
R_RELATION = r'</?(?P<target_IRI>[^>]+)>; rel="(?P<relation>first|previous|next|last)"'
```
That way it will also work when `/` isn't there. (Also the `\` shouldn't be necessary.)

Reply:
Thanks for your review. I updated with the suggested change 🙂